src/screens/TopicPageScreen/index.js

```diff
@@ -390,10 +395,17 @@ const TopicPageScreen = (props) => {
     />
   );
 
-  if (isInitialLoading) return null;
   return (
-    <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
+    <>
       <NavHeader
+        initialData={{
+          channelPicutre: params.channelPicture,
+          coverImage: params.coverImage,
+          isFollowing: params.isFollowing,
+          memberCount: params.memberCount
+        }}
+        isLoading={isInitialLoading}
+        opacityImage={opacityImage}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for rendering null when `isInitialLoading` is true has been removed. Instead, `NavHeader` is now rendered with `isLoading={isInitialLoading}`. Ensure that the `NavHeader` component handles the `isLoading` prop correctly to prevent unwanted behavior during loading states.
```diff
- if (isInitialLoading) return null;
+ <NavHeader
+   initialData={{
+     channelPicutre: params.channelPicture,
+     coverImage: params.coverImage,
+     isFollowing: params.isFollowing,
+     memberCount: params.memberCount
+   }}
+   isLoading={isInitialLoading}
+   opacityImage={opacityImage}
+   domain={topicName}
+   animatedHeight={headerHeight}
+   onShareCommunity={onShareCommunity}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->